website/src/components/Messages/RenderedMarkdown.tsx

```diff
@@ -163,12 +163,7 @@ const RenderedMarkdown = ({ markdown }: RenderedMarkdownProps) => {
 const MemorizedMarkdown = memo(function MemorizedMarkdown(props: ReactMarkdownOptions) {
   return (
     <Prose as="div" sx={sx}>
-      <ReactMarkdown
-        {...props}
-        disallowedElements={disallowedElements}
```

Comment:
we use this to render the markdown in the tasks and messages input by users as well. We intentionally disabled images because people would insert "bad" images.

This needs to stay in.

Reply:
Ok, and how to make it work for plugins, that responds with images?

Reply:
the disabled elements should be a parameter, and you can override it for the chat case.

Reply:
Ok